keras/src/losses/losses_test.py

```diff
@@ -1181,6 +1181,14 @@ def test_ignore_class(self):
         loss = cce_obj(y_true, logits)
         self.assertAllClose([[0.0, 1.48012]], loss, 3)
 
+        y_true = np.array([[[-1], [2]]])
+        logits = np.array([[[0.854, 0.698, 0.598], [0.088, 0.86, 0.018]]])
+        cce_obj = losses.SparseCategoricalCrossentropy(
+            from_logits=True, ignore_class=-1, reduction=None
+        )
+        loss = cce_obj(y_true, logits)
+        self.assertAllClose([[0.0, 1.48012]], loss, 3)
```

Comment:
Is the `3` for `atol`? That would render the check ineffective.

Reply:
Yes, and a lot of other tests use 3 like that, should we change them to a more precise one like 0.00001?

Reply:
That's not normal, what other tests are using 3?

You can just not pass any value, it will default to 1e-6 which is a good value to use in general.

Reply:
Fixed.